app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>here are individual users</h1>
\ No newline at end of file
+<div id="show_container_user">
+  <div id="container_container">
+      <img id="container_container_img" src="<%= @user.Photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= @user.Name %></h3>
+        <p>Number of posts: <%= @user.PostsCounter %></p>
+      </div>
+    </div>
+  <div id="show_container_user_bio">
+    <h4>Bio</h4>
+    <p>
+      Lorem ipsum dolor sit amet, 
+      consectetur adipiscing elit, 
+      sed do eiusmod tempor incididunt ut 
+      labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+      laboris nisi ut aliquip ex ea commodo consequat.
+    </p>
+  </div>
+  <% @posts.each do |post|%>
```

Comment:
- [ ] We really need to render only the 3 most recent posts on each user show page, not all the posts. You had a method that does that in your user model **_Hints_**:

` <% @user.recent_three_posts.each do |post|%>`